crates/holochain_zome_types/src/lib.rs

```diff
@@ -78,9 +78,6 @@ pub mod fixt;
 #[cfg(feature = "test_utils")]
 pub mod test_utils;
 
-#[cfg(all(any(test, feature = "test_utils"), feature = "fuzzing"))]
-pub mod entropy;
```

Comment:
Valid to remove, but unrelated. Would prefer this on a separate PR in future, or at a comment calling out why it's been done. Just makes it easier for reviewers who didn't happen to be involved in the conversation about this.